src/SDKs/_metadata/monitor_resource-manager.txt

```diff
@@ -0,0 +1,12 @@
+2019-08-01 03:08:36 UTC
```

Comment:
@gucalder, this file should be under **eng\mgmt\mgmtmetadata**

Reply:
Ok, moving the file.
BTW: this file was created automatically. There has to be some kind of bug or I am running the wrong command to generate the code.

Reply:
@gucalder , you need to first pull latest code from master, then follow the step 2 https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec
**msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules**